apps/guide/src/content/02-installations-and-preparations/04-adding-your-bot-to-servers.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Adding your bot to servers
+category: Installations and preparations
+---
+
+# Adding your bot to servers
+
+After you [set up a bot application](./setting-up-a-bot-application), you'll notice that it's not in any servers yet. So how does that work?
+
+Before you're able to see your bot in your own (or other) servers, you'll need to add it by creating and using a unique invite link using your bot application's client id.
+
+## Bot invite links
+
+The basic version of one such link looks like this:
+
+<CH.Code lineNumbers={false}>
+
+```
+https://discord.com/api/oauth2/authorize?client_id=123456789012345678&permissions=0&scope=bot%20applications.commands
+```
+
+</CH.Code>
+
+The structure of the URL is quite simple:
+
+- _`https://discord.com/api/oauth2/authorize`_ is Discord's standard structure for authorizing an OAuth2 application (such as your bot application) for entry to a Discord server.
+- _`client_id=...`_ is to specify _which_ application you want to authorize. You'll need to replace this part with your client's id to create a valid invite link.
+- _`permissions=...`_ describes what permissions your bot will have on the server you are adding it to.
+- _`scope=bot%20applications.commands`_ specifies that you want to add this application as a Discord bot, with the ability to create slash commands.
+
+<Alert title="Warning" type="warning">
+	If you get an error message saying "Bot requires a code grant", head over to your application's settings and disable
+	the "Require OAuth2 Code Grant" option. You shouldn't enable this option unless you know why you need to.
+</Alert>
+
+## Creating and using your invite link
```

Comment:
Can we add a "Generating an invite URL programmatically" section [like in #9412](https://github.com/discordjs/discord.js/pull/9412/files#diff-7a82238d76ba72fff2554ad5afe8c4e9ed5f86493506e41ed42d187baabb299aR42)?

Reply:
This page is positioned during the initial configuration, adding your bot to a server for the very first time and getting it running. I don't believe programmatically generating invite URLs aligns with the very beginner nature of this section of the guide.